<launch>
  <node name="ps3_cam1_image_publisher" pkg="ps3_image_publisher" type='image_publisher.py'>
      <param name="usb_port" value="0-8" />
      <param name="fps" value="50" />
      <param name="exposure" value="10" />
      <param name="gain" value="20" />
      <remap from='/camera/cameraInfo' to='/mono_camera/cameraInfo' />
      <remap from='/camera/image_raw' to='/mono_camera/image_raw' />
  </node>
</launch>
